core/common/assets/js/api/index.js

```diff
@@ -17,6 +17,12 @@ import Shortcuts from './core/shortcuts';
 import * as hookData from './modules/hooks/data/';
 import * as hookUI from './modules/hooks/ui';
 
+/**
+ * @typedef {{}} Args
+ * @property {Container} container
+ * @property {Array.<Container>} containers
+ */
+
```

Comment:
Revert this change api/index should not know about the existence of container(s).